app/views/ProjectEdit/Framework/index.tsx

```diff
@@ -225,13 +224,13 @@ function ProjectFramework(props: Props) {
 
     const handleNewFrameworkAddSuccess = useCallback((newFrameworkId: number) => {
         setSelectedFramework(newFrameworkId);
-        retriggerGetFrameworkListRequest();
+        setFrameworkList([]);
+        setOffset(0);
         hideFrameworkAddModal();
-    }, [setSelectedFramework, hideFrameworkAddModal, retriggerGetFrameworkListRequest]);
+    }, [setSelectedFramework, hideFrameworkAddModal]);
```

Comment:
Why not retrigger?

Reply:
change in offset auto retriggers framework pull

Reply:
omg, but what if offset is already 0?

Reply:
good point :P